.circleci/config.yml

```diff
@@ -76,6 +76,30 @@ jobs:
             . venv/bin/activate
             pytest -v
 
+  python37:
+    docker:
+      - image: circleci/python:3.7.9
+      - image: circleci/postgres:9.6.5-alpine-ram
+      - image: circleci/mariadb:10-ram
+      - image: circleci/redis:5.0.4
+      - image: rabbitmq:3.5.4
+      - image: circleci/mongo:4.2.3-ram
+      - image: singularities/pubsub-emulator
+        environment:
+          PUBSUB_PROJECT_ID: "project-test"
+          PUBSUB_LISTEN_ADDRESS: "0.0.0.0:8432"
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - pip-install-deps
+      - run:
+          name: run tests
+          environment:
+            INSTANA_TEST: "true"
+          command: |
+            . venv/bin/activate
+            pytest -v
+
```

Comment:
Let's see if we can run `asqynp` on Python 3.8 - if not, then we can have this additional test job.

Reply:
Looks like support only until Python 3.7: https://github.com/benjamin-hodgson/asynqp/pull/104 
We can keep it @pdimitra